Tests/Functional/Controller/Fixtures/Database/FrontEndEditorController/FrontendUsers.csv

```diff
@@ -0,0 +1,3 @@
+"fe_users"
+,"uid","pid","username","password"
```

Comment:
We can drop the `password` column from this fixture file as it's not relevant for these tests.